aws/app/lambda/submission/submission.js

```diff
@@ -57,14 +57,13 @@ const sendData = async (submissionID) => {
 };
 
 const saveData = async (submissionID, formData) => {
-  const securityAttribute = formData.securityAttribute;
+  const securityAttribute = formData.securityAttribute ?? "Unclassified";
```

Comment:
Looks like this is what makes the submission lambda crash. We use NodeJS 14.x in AWS and the `??` operator has been introduced in version 15+

Reply:
You can use `const securityAttribute = formData.securityAttribute ? formData.securityAttribute : "Unclassified";` instead

Reply:
Weird.. here it says it's compatible with NodeJS 14: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator#browser_compatibility

Reply:
@bryan-robitaille I have seen that page but then I found https://node.green/#ES2021-features-Logical-Assignment-----basic-support

Reply:
It might be something else then. I will investigate

Reply:
I was testing this locally and noticed some of the local lambdas were running on NodeJS 12

Reply:
Yep that was it! Works fine on NodeJS 14. Will make a PR to make sure our local lambdas use NodeJS 14. cc @bryan-robitaille @falila